build/csv/csv-gen.sh

```diff
@@ -37,7 +37,14 @@ function generate_csv() {
 
     "${YQ_CMD_MERGE[@]}" "$CSV_FILE_NAME" "$ASSEMBLE_FILE_OCP"
 
+    # We don't need to include these files in csv as ocs-operator creates its own.
+    rm -rf "../../build/csv/ceph/$PLATFORM/manifests/rook-ceph-operator-config_v1_configmap.yaml"
+    rm -rf "../../build/csv/ceph/$PLATFORM/manifests/objectbucket.io_objectbucketclaims.yaml" 2>/dev/null || true
+    rm -rf "../../build/csv/ceph/$PLATFORM/manifests/objectbucket.io_objectbuckets.yaml" 2>/dev/null || true
```

Comment:
nit: I think the ` || true` in this case is not needed as an `rm -f` is used. So even if the file doesn't exit the cmd will succeed.

Reply:
removed.